saleor/graphql/payment/tests/mutations/test_transaction_request_action.py

```diff
@@ -573,3 +583,59 @@ def test_transaction_request_action_missing_event(
     )
 
     assert mocked_is_active.called
+
+
+@patch("saleor.plugins.manager.PluginsManager.is_event_active_for_any_plugin")
+def test_transaction_request_action_missing_event_refund_requested(
```

Comment:
I think that you could extend above test (test_transaction_request_action_missing_event) to confirm that we didn't create an REQUEST event.